+++
# Recent Publications widget.
# This widget displays recent publications from `content/publication/`.
widget = "custom"
active = true
date = 2016-04-20

title = "Publications"
subtitle = ""

# Order that this section will appear in.
weight = 30

+++

## Book
* Albert, I., Ancelet, S., David, O., Denis, J.-B., Makowski, D., Parent, &Eacute;, **Rau, A.**, and Soubeyrand, S. (2015). *Initiation &agrave; la statistique bay&eacute;sienne : Bases th&eacute;oriques et applications en alimentation, environnmenet, &eacute;pid&eacute;miologie et g&eacute;n&eacute;tique*. &Eacute;ditions Ellipses, collection r&eacute;f&eacute;rences sciences. 
<a class="btn btn-primary btn-outline btn-xs" href="http://maiage.jouy.inra.fr/?q=fr/bbb.accueil.html" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://www.editions-ellipses.fr/product_info.php?products_id=10293" target="_blank" rel="noopener">
  Publisher link
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311814" target="_blank" rel="noopener">
  ProdInra
</a>


## Dissertations
* **Rau, A.** (2017) Statistical methods and software for the analysis of transcriptomic data. HDR (Habilitation &agrave; diriger des recherches) thesis, Universit&eacute; d'&Eacute;vry Val-d'Essonne.
<a class="btn btn-primary btn-outline btn-xs" href="http://prodinra.inra.fr/ft?id=2922C029-96B2-4A41-9B9B-6FB981ABADA6" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://speakerdeck.com/andreamrau/hdr-defense-presentation" target="_blank" rel="noopener">
  Slides
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/408838" target="_blank" rel="noopener">
  ProdInra
</a>  
*Note: HDR is a high-level (post-PhD) degree granted by French universities that provides an accreditation to supervise research.*

* **Rau, A.** (2010) Reverse engineering gene networks using genomic time-course data. PhD thesis, Purdue University.
<a class="btn btn-primary btn-outline btn-xs" href="https://docs.lib.purdue.edu/dissertations/AAI3444841/" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/189250" target="_blank" rel="noopener">
  ProdInra
</a>

## Journal Articles

<center>**2020**</center>


* Sellem, E., Marthey, S., **Rau, A.**, Jouneau, L., Bonnet, A., Perrier, J.-P., Fritz, S., Le Danvic, C. Boussaha, M., Kiefer, H., Jammes, H., Schiblier, L. (2020) A comprehensive overview of bull sperm-borne small non-coding RNAs and their diversity in six breeds. *Epigenetics and Chromatin* 13:19. doi: 10.1186/s13072-020-00340-0.  
<a class="btn btn-primary btn-outline btn-xs" href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00340-0" target="_blank" rel="noopener">
  PDF
</a>

* **Rau, A.**, Manansala, R., Flister, M. J., Rui, H., Jaffr&eacute;zic, F., Lalo&euml;, D.&sup1;, and Auer, P. L.&sup1; (2020) Individualized multi-omic pathway deviation scores using multiple factor analysis. *Biostatistics* (accepted), https://doi.org/10.1101/827022.   
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href=" 	https://bioconductor.org/packages/padma/" target="_blank" rel="noopener">
  Software
</a>  
&sup1;These authors contributed equally to this work.

* Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. *Annals of Applied Statistics*, 14:2, 752-767.
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01916941" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://projecteuclid.org/euclid.aoas/1593449324" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/441070" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2019**</center>

* Jehl, F., D&eacute;sert, C., Klopp, C., Brenet, M., **Rau, A.**,  Leroux, S., Boutin, M., Muret, K.,  Blum, Y., Esquerr&eacute;, D., Gourichon, D., Burlot, T., Collin, A., Pitel, F., Benani, A., Zerjal, T., Lagarrigue, S. (2019) Chicken adaptive response to low energy diet: main role of the hypothalamic lipid metabolism revealed by a phenotypic and multi-tissue transcriptomic approach. *BMC Genomics*, 20. doi: 10.1186/s12864-019-6384-8.  
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6384-8" target="_blank" rel="noopener">
  PDF
</a>


* Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., **Rau, A.**, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. *BMC Biology* 17: 108. 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/441070" target="_blank" rel="noopener">
  ProdInra
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://www.fragencode.org" target="_blank" rel="noopener">
  Website
</a>


* Dhara, S., **Rau, A.**, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. *Scientific Reports* 9:14198, doi: 10.1038/s41598-019-50485-6.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener">
 Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
 Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481438" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Dhara, S., Udvadia, A., and Auer, P. (2019) *Regeneration Rosetta*: An interactive web application to explore
regeneration-associated gene expression and chromatin accessibility. *G3: Genes|Genomes|Genetics*, doi: 10.1534/g3.119.400729..  
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
 Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481437" target="_blank" rel="noopener">
  ProdInra
</a>


* Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., **Rau, A.**, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. *G3: Genes|Genomes|Genetics* 9(5):1739-1743. doi: 10.1534/g3.118.200873.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.g3journal.org/content/9/5/1739" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/473094" target="_blank" rel="noopener">
  ProdInra
</a>

* Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., Estell&eacute;, J. Popova, M., Pons, N., Bellot, P., Mach, N., **Rau, A.**, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. *Journal of Animal Breeding and Genetics*, doi: 10.1111/jbg.12427.
<a class="btn btn-primary btn-outline btn-xs" href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbg.12427" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="	
https://prodinra.inra.fr/record/481441" target="_blank" rel="noopener">
  ProdInra
</a>



<center>**2018**</center>

* **Rau, A.**, Flister, M. J., Rui, H. and Livermore Auer, P. (2018) Exploring drivers of gene expression in The Cancer Genome Atlas. *Bioinformatics*, bty551, doi: https://doi.org/10.1101/227926.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener">
  Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/452734" target="_blank" rel="noopener">
  ProdInra
</a>


* Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. *Journal of Applied Statistics*, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481439" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.** and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. *Briefings in Bioinformatics*, bbw128, https://doi.org/10.1093/bib/bbw128.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/413088" target="_blank" rel="noopener">
  ProdInra
</a>

* Verrier, E., Genet, C., Lalo&euml;, D., Jaffr&eacute;zic, J.,  **Rau, A.**, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. *BMC Genomics*, 19:482.
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/434908" target="_blank" rel="noopener">
  ProdInra
</a>

* Maroilley, T., Berri, M., Lemonnier, G., Esquerr&eacute;, D., Chevaleyre, C., M&eacute;lo, S., Meurens, F., Coville, J.L., Leplat, J.J, **Rau, A.**, Bed'hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and Estell&eacute;, J. (2018). Immunome differences between porcine ileal and jejunal Peyer's patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. *Scientific Reports*, 8:9077. 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41598-018-27019-7" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/413088" target="_blank" rel="noopener">
  ProdInra
</a>

* Mondet, F., **Rau, A.**, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018). Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. *BMC Genomics*, 19:328.
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/432918" target="_blank" rel="noopener">
  ProdInra
</a>

* He, B., Tjhung, K., Bennett, N., Chou, Y., **Rau, A.**, Huang, J., and Derda, R. (2018). Compositional bias in na&iuml;ve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. *Scientific Reports*, 8:1214.
<a class="btn btn-primary btn-outline btn-xs" href="http://rdcu.be/E4Zi" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/421248" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2017**</center>

* Monneret, G., Jaffr&eacute;zic, F., **Rau, A.**, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. *PLoS One* 12(3): e0171142.
<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/Monneret/MarginalCausality" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/391068" target="_blank" rel="noopener">
  ProdInra
</a>

* Sauvage, C., **Rau, A.**, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Gl&eacute;min, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. *The Plant Journal* 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/393545" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2016**</center>


* Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., **Rau, A.**, Rogier, O., Caius, J.,  Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. *Briefings in Bioinformatics*, doi: https://doi.org/10.1093/bib/bbw092.
<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/377567" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2015**</center>


* Gallopin, M., Celeux, G., Jaffr&eacute;zic, F., **Rau, A.** (2015) A model selection criterion for model-based clustering of annotated gene expression data. *Statistical Applications in Genetics and Molecular Biology*, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/336851" target="_blank" rel="noopener">
  ProdInra
</a>

* Monneret, G., Jaffr&eacute;zic, F., **Rau, A.**, Nuel, G. (2015) Estimation d'effets causaux dans les r&eacute;seaux de r&eacute;gulation g&eacute;nique : vers la grande dimension. *Revue d'intelligence artificielle*, 29(2): 205-227.
<a class="btn btn-primary btn-outline btn-xs" href="http://ria.revuesonline.com/article.jsp?articleId=20894" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311809" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. *Bioinformatics*, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener">
  Code
</a>
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311812" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2014**</center>


* **Rau, A.**, Marot, G. and Jaffr&eacute;zic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. *BMC Bioinformatics*, 15:91.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.biomedcentral.com/1471-2105/15/91/abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/257627" target="_blank" rel="noopener">
  ProdInra
</a>

* Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., **Rau, A.**, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. *PLoS One*, 9(12): e114598.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.plosone.org/article/info\%3Adoi\%2F10.1371\%2Fjournal.pone.0114598" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/289922" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2013**</center>

* Nuel, G., **Rau, A.**, and Jaffr&eacute;zic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments. *Quality Technology and Quantitative Management* 11(1):23-37.
<a class="btn btn-primary btn-outline btn-xs" href="http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/283565" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Jaffr&eacute;zic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. *BMC Systems Biology* 7:111.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.biomedcentral.com/1752-0509/7/111/abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/255885" target="_blank" rel="noopener">
  ProdInra
</a>

* Gallopin, M. **Rau, A.**, and Jaffr&eacute;zic, F. (2013). A hierarchical Poisson log-normal model for network inference from RNA sequencing data. *PLoS One* 8(10): e77503.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/221846" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Gallopin, M., Celeux, G., and Jaffr&eacute;zic, F. (2013). Data-based filtering for replicated high-throughput transcriptome sequencing experiments. *Bioinformatics* 29(17): 2146-2152.
<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/29/17/2146" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/201701" target="_blank" rel="noopener">
  ProdInra
</a>

* Dillies, M.-A.&sup1;, **Rau, A.&sup1;**, Aubert, J.&sup1;, Hennequet-Antier, C.&sup1;, Jeanmougin, M.&sup1;, Servant, N.&sup1;, Keime, C.&sup1;, Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., Lalo&euml;, D., Le Gall, C., Scha&euml;ffer, B., Charif, D., Le Crom, S.&sup1;, Guedj, M.&sup1;, and Jaffr&eacute;zic, F&sup1;. (2013). A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. *Briefings in Bioinformatics*, doi:10.1093/bib/bbs046.
<a class="btn btn-primary btn-outline btn-xs" href="http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/173448" target="_blank" rel="noopener">
  ProdInra
</a>  
&sup1;These authors contributed equally to this work.

* Brenault, P., Lefevre, L. **Rau, A.**, Lalo&euml;, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. *Veterinary Research*, 45:16.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.veterinaryresearch.org/content/45/1/16" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/263994" target="_blank" rel="noopener">
  ProdInra
</a>

<center>**2012 and before**</center>


* **Rau, A.**, Jaffr&eacute;zic, F., Foulley, J.-L., and Doerge, R. W. (2012). Reverse engineering gene regulatory networks using approximate Bayesian computation. *Statistics and Computing*, 22: 1257-1271.
<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1109.1402" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://link.springer.com/article/10.1007\%2Fs11222-011-9309-1" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/172363" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Jaffr&eacute;zic, F., Foulley, J.-L., and Doerge, R. W. (2010). An empirical Bayesian method for estimating biological networks from temporal microarray data. *Statistical Applications in Genetics and Molecular Biology*: Vol. 9: Iss. 1, Article 9.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/ebdbNet" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/6927" target="_blank" rel="noopener">
  ProdInra
</a>

* Furth, A., Mandrekar, S., Tan, A. **Rau, A.**, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008). A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. *Cancer Chemotherapy Pharmacology*, 61(1): 39-45.
</a>
<a class="btn btn-primary btn-outline btn-xs" href="	
https://prodinra.inra.fr/record/189243" target="_blank" rel="noopener">
  ProdInra
</a>  


## Book chapters
* Martin-Magniette, M.-L., Maugis-Rabusseau, C. and **Rau, A.** (2017) Clustering of co-expressed genes. In: *Model Choice and Model Aggregation*. Ed. F. Bertrand, J.-J. Droesbeke, G. Saporta, C. Thomas-Agnan.
<a class="btn btn-primary btn-outline btn-xs" href="http://www.editionstechnip.com/fr/catalogue-detail/2247/model-choice-and-model-aggregation.html" target="_blank" rel="noopener">
  Publisher link
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01943296" target="_blank" rel="noopener">
  HAL
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481436" target="_blank" rel="noopener">
  ProdInra
</a>  

## Submitted works


* Revilla, M., **Rau, A.**, Crespo-Piazuelo, D., Ramayo-Caldas, Y., Estell&eacute;, J., INIA, Ballester, M., Folch, J. M. (2019) An integrative gene network analysis of the genetic determination of pig fatty acid composition based on adipose tissue RNA sequencing. *Submitted.*

* Devogel, N., Auer, P. L., Manansala, R., **Rau, A.**, and Wang, T. (2020) A unified linear mixed model for familial relatedness and population structure in genetic association studies. *Submitted.*

* Cho, Y., **Rau, A.**, Reiner, A., Auer, P. L. (2020) Mendelian randomization analysis with survival outcomes. *Submitted.*

* Cazals, A., Estell&eacute;, J., Bruneau, N., Coville, J.-L., Menanteau, P., Rossignol, M.-N., Jardet, D., Bevilacqua, C., **Rau, A.**, Bed'Hom, B., Velge, P., and Calenge, F. (2020) Impact of host genetics on caecal microbiota composition and on *Salmonella* carriage in chicken. *Submitted.*

* Karlis, D., Mazo, G. and **Rau, A.** (2020) A randomized pairwise likelihood method with applications to multivariate count data. *In preparation.*
