+++
# Recent Publications widget.
# This widget displays recent publications from `content/publication/`.
widget = "custom"
active = true
date = 2016-04-20

title = "Publications"
subtitle = ""

# Order that this section will appear in.
weight = 30

+++

## Dissertations, books & book chapters
1. Duranthon, V., Araújo, S., Palma, M., **Rau, A.**, Matzapetakis, M., and Almeida, A. (2021) Rabbit research in the post-genomic era: transcriptome, proteome, and metabolome analysis. *In: The Genetics and Genomics of the Rabbit.*, Ed. L. Fontanesi.
     
1. **Rau, A.** (2017) Statistical methods and software for the analysis of transcriptomic data. *HDR thesis*, Université d’Évry Val-d’Essonne.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/tel-02786130" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/tel-02786130/document" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://speakerdeck.com/andreamrau/hdr-defense-presentation" target="_blank" rel="noopener"> 
Slides
</a>   
1. Martin-Magniette, M.-L., Maugis-Rabusseau, C. and **Rau, A.** (2017) Clustering of co-expressed genes. *In: Model Choice and Model Aggregation*, Ed. F. Bertrand, J.-J. Droesbeke, G. Saporta, C. Thomas-Agnan.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02789928" target="_blank" rel="noopener"> 
HAL 
</a>      <a class="btn btn-primary btn-outline btn-xs"   href="http://www.editionstechnip.com/fr/catalogue-detail/2247/model-choice-and-model-aggregation.html" target="_blank" rel="noopener"> 
Publisher link 
</a>  
1. Albert, I., Ancelet, S., David, O., Denis, J.-B., Makowski, D., Parent, É., **Rau, A.**, and Soubeyrand, S. (2015) Initiation à la statistique bayésienne : Bases théoriques et applications en alimentation, environnement, épidémiologie et génétique. *Éditions Ellipses*, collection références sciences.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194144" target="_blank" rel="noopener"> 
HAL 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="http://maiage.jouy.inra.fr/?q=fr/bbb.accueil.html" target="_blank" rel="noopener"> 
Code 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://www.editions-ellipses.fr/product_info.php?products_id=10293" target="_blank" rel="noopener"> 
Publisher link 
</a>  
1. **Rau, A.** (2010) Reverse engineering gene networks using genomic time-course data.. *PhD thesis*, Purdue University.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/tel-02819388" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://docs.lib.purdue.edu/dissertations/AAI3444841/" target="_blank" rel="noopener"> 
PDF 
</a>      

## Peer-reviewed publications
1. Sellem, E., Marthey, S., **Rau, A.**, Jouneau, L., Bonnet, A., Le Danvic, C., Kiefer, H., Jammes, H., and Schibler, L. (2021) Dynamics of cattle sperm sncRNAs during maturation, from testis to ejaculated sperm. *Epigenetics and Chromatin*, to appear.
      
1. Mach, N., Moroldo, M., **Rau, A.**, Lecardonnel, J., Le Moyec, L., Robert, C., and Barrey, E. (2021) Understanding the holobiont: crosstalk between gut microbiota and mitochondria during endurance. *Frontiers Molecular Biosciences*, 8:656204.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03197210" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.frontiersin.org/articles/10.3389/fmolb.2021.656204/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Molecular_Biosciences&id=656204" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/2021.01.08.425889v1" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. Devogel, N., Auer, P. L., Manansala, R., **Rau, A.**, and Wang, T. (2020) A unified linear mixed model for familial relatedness and population structure in genetic association studies. *Genetic Epidemiology*, 45(3): 305-315.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176147v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/10.1002/gepi.22371" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Cho, Y., **Rau, A.**, Reiner, A., Auer, P. L. (2020) Mendelian randomization analysis with survival outcomes. *Genetic Epidemiology*, 45(1): 16-23.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03155097v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.22354" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Sellem, E., Marthey, S., **Rau, A.**, Jouneau, L., Bonnet, A., Perrier, J.-P., Fritz, S., Le Danvic, C. Boussaha, M., Kiefer, H., Jammes, H., Schiblier, L. (2020) A comprehensive overview of bull sperm-borne small non-coding RNAs and their diversity in six breeds. *Epigenetics and Chromatin*, 13:19.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176141v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00340-0" target="_blank" rel="noopener"> 
PDF 
</a>       
1. **Rau, A.**, Manansala, R., Flister, M. J., Rui, H., Jaffrézic, F., Laloë, D., and Auer, P. L. (2020) Individualized multi-omic pathway deviation scores using multiple factor analysis. *Biostatistics*, kxaa029.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02968360v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/biostatistics/advance-article/doi/10.1093/biostatistics/kxaa029/5882080?guestAccessKey=9a15cdfb-100e-4478-8b68-e0a1f9e495f4" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/padma/" target="_blank" rel="noopener"> 
Software 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener"> 
Code 
</a>    
1. Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. *Annals of Applied Statistics*, 14:2, 752-767.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789493" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://projecteuclid.org/euclid.aoas/1593449324" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener"> 
Code 
</a>    
1. Jehl, F., Désert, C., Klopp, C., Brenet, M., **Rau, A.**, Leroux, S., Boutin, M., Muret, K., Blum, Y., Esquerré, D., Gourichon, D., Burlot, T., Collin, A., Pitel, F., Benani, A., Zerjal, T., Lagarrigue, S. (2019) Chicken adaptive response to low energy diet: main role of the hypothalamic lipid metabolism revealed by a phenotypic and multi-tissue transcriptomic approach. *BMC Genomics*, 20.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02463367v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6384-8" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., **Rau, A.**, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. *BMC Biology*, 18:48.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02437599v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener"> 
Preprint 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="http://www.fragencode.org/" target="_blank" rel="noopener"> 
Website 
</a>  
1. Dhara, S., **Rau, A.**, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. *Scientific Reports*, 9:14198.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02791772" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
1. **Rau, A.**, Dhara, S., Udvadia, A., and Auer, P. (2019) Regeneration Rosetta: An interactive web application to explore regeneration-associated gene expression and chromatin accessibility. *G3: Genes|Genomes|Genetics*, 9(12): 3953-3959.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789080" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
1. Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., **Rau, A.**, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. *G3: Genes|Genomes|Genetics*, 9(5): 1739-1743.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02626057" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/5/1739" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., Estellé, J. Popova, M., Pons, N., Bellot, P., Mach, N., **Rau, A.**, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. *Journal of Animal Breeding and Genetics*, 137:49-59.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02495627" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbg.12427" target="_blank" rel="noopener"> 
PDF 
</a>       
1. **Rau, A.**, Flister, M. J., Rui, H. and Livermore Auer, P. (2019) Exploring drivers of gene expression in The Cancer Genome Atlas. *Bioinformatics*, 35(1): 62-68.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02625326" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
1. Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. *Journal of Applied Statistics*, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02789493" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener"> 
Software 
</a>     
1. **Rau, A.** and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. *Briefings in Bioinformatics*, bbw128.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02624483" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener"> 
Software 
</a>     
1. Verrier, E., Genet, C., Laloë, D., Jaffrézic, J., **Rau, A.**, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. *BMC Genomics*, 19:482.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02444518" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Maroilley, T., Berri, M., Lemonnier, G., Esquerré, D., Chevaleyre, C., Mélo, S., Meurens, F., Coville, J.L., Leplat, J.J, **Rau, A.**, Bed’hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and Estellé, J. (2018) Immunome differences between porcine ileal and jejunal Peyer’s patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. *Scientific Reports*, 8:9077.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02356215" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-018-27019-7" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Mondet, F., **Rau, A.**, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018) Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. *BMC Genomics*, 19:328.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/inserm-02063466" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener"> 
PDF 
</a>       
1. He, B., Tjhung, K., Bennett, N., Chou, Y., **Rau, A.**, Huang, J., and Derda, R. (2018) Compositional bias in naïve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. *Scientific Reports*, 8:1214.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02623801" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://rdcu.be/E4Zi" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Monneret, G., Jaffrézic, F., **Rau, A.**, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. *PLoS One*, 12(3): e0171142.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01504607" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Monneret/MarginalCausality" target="_blank" rel="noopener"> 
Code 
</a>    
1. Sauvage, C., **Rau, A.**, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Glémin, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. *The Plant Journal*, 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01594937" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., **Rau, A.**, Rogier, O., Caius, J., Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. *Briefings in Bioinformatics*, 19(1):65-76.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01595551" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Gallopin, M., Celeux, G., Jaffrézic, F., **Rau, A.** (2015) A model selection criterion for model-based clustering of annotated gene expression data. *Statistical Applications in Genetics and Molecular Biology*, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01255908" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener"> 
Code 
</a>    
1. Monneret, G., Jaffrézic, F., **Rau, A.**, Nuel, G. (2015) Estimation d’effets causaux dans les réseaux de régulation génique : vers la grande dimension. *Revue d’intelligence artificielle*, 29(2): 205-227.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194142" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ria.revuesonline.com/article.jsp?articleId=20894" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener"> 
Code 
</a>    
1. **Rau, A.**, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. *Bioinformatics*, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01108821" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener"> 
Software 
</a>     
1. **Rau, A.**, Marot, G. and Jaffrézic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. *BMC Bioinformatics*, 16:31.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00978902" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1471-2105/15/91/abstract" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank" rel="noopener"> 
Software 
</a>     
1. Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., **Rau, A.**, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. *PLoS One*, 9(12): e114598.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194096" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info/%3Adoi/%2F10.1371/%2Fjournal.pone.0114598" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Nuel, G., **Rau, A.**, and Jaffrézic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments.. *Quality Technology and Quantitative Management*, 11(1):23-37.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194064" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf" target="_blank" rel="noopener"> 
PDF 
</a>       
1. **Rau, A.**, Jaffrézic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. *BMC Systems Biology*, 8:51.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193793" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1752-0509/7/111/abstract" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener"> 
Code 
</a>    
1. Gallopin, M. **Rau, A.**, and Jaffrézic, F. (2013) A hierarchical Poisson log-normal model for network inference from RNA sequencing data. *PLoS One*, 8(10): e77503.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01004715" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503" target="_blank" rel="noopener"> 
PDF 
</a>       
1. **Rau, A.**, Gallopin, M., Celeux, G., and Jaffrézic, F. (2013) Data-based filtering for replicated high-throughput transcriptome sequencing experiments. *Bioinformatics*, 29(17): 2146-2152.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00927025" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/29/17/2146" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank" rel="noopener"> 
Software 
</a>     
1. Dillies, M.-A., **Rau, A.**, Aubert, J., Hennequet-Antier, C., Jeanmougin, M., Servant, N., Keime, C., Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., Laloë, D., Le Gall, C., Schaëffer, B., Charif, D., Le Crom, S., Guedj, M., and Jaffrézic, F. (2013) A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. *Briefings in Bioinformatics*, 14(6): 671-683.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00782486" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Brenault, P., Lefevre, L. **Rau, A.**, Laloë, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. *Veterinary Research*, 45:16.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193839" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.veterinaryresearch.org/content/45/1/16" target="_blank" rel="noopener"> 
PDF 
</a>       
1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2012) Reverse engineering gene regulatory networks using approximate Bayesian computation. *Statistics and Computing*, 22: 1257-1271.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01019032" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://link.springer.com/article/10.1007/%2Fs11222-011-9309-1" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1109.1402" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) An empirical Bayesian method for estimating biological networks from temporal microarray data. *Statistical Applications in Genetics and Molecular Biology*, 9(1): 9.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193424" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/ebdbNet" target="_blank" rel="noopener"> 
Software 
</a>     
1. Furth, A., Mandrekar, S., Tan, A. **Rau, A.**, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008) A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. *Cancer Chemotherapy Pharmacology*, 61(1): 39-45.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02664062" target="_blank" rel="noopener"> 
HAL 
</a>        

## Pre-prints, technical reports, and other publications
1. **Rau, A.** (2021) Cooking up knowledge from big data using data science. *Frontiers in Young Minds*, 9:632923.
 <a class="btn btn-primary btn-outline btn-xs"   href="https://kids.frontiersin.org/article/10.3389/frym.2021.632923" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://dx.doi.org/10.3389/frym.2021.632923" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. Mollandin, F., **Rau, A.**, and Croiseau, P. (2021) An evaluation of the interpretability and predictive performance of the BayesR model for genomic prediction., Submitted.
  <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/2020.10.23.351700v1" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. Mazo, G., Karlis, D., and **Rau, A.** (2021) A randomized pairwise likelihood method for complex statistical inferences., Submitted.
  <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.archives-ouvertes.fr/hal-03126621" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. Cazals, A., Estellé, J., Bruneau, N., Coville, J.-L., Menanteau, P., Rossignol, M.-N., Jardet, D., Bevilacqua, C., **Rau, A.**, Bed’Hom, B., Velge, P., and Calenge, F. (2020) Impact of host genetics on caecal microbiota composition and on Salmonella carriage in chicken., Submitted.
  <a class="btn btn-primary btn-outline btn-xs"   href="https://www.researchsquare.com/article/rs-76645/v1" target="_blank" rel="noopener"> 
Preprint 
</a>      
1. Bruford, M., Leroy, G., Orozco-terWengel, P., **Rau, A.**, and Simianer H. (2015) Section B: Molecular tools for exploring genetic diversity. *The Second Report on the State of the World’s Animal Genetic Resources for Food and Agriculture*, FAO Commission on Genetic Resources for Food and Agriculture.
 <a class="btn btn-primary btn-outline btn-xs"   href="http://www.fao.org/3/a-i4787e.pdf" target="_blank" rel="noopener"> 
PDF 
</a>       
1. Nuel, G., **Rau, A.**, and Jaffrézic, F. (2013) Joint likelihood calculation for intervention and observational data from a Gaussian Bayesian network. *arXiv*, 1305.0709.
      
1. **Rau, A.**, Celeux, G., Martin-Magniette, M.-L., and Maugis-Rabusseau, C. (2011) Clustering high-throughput sequencing data with Poisson mixture models. *Inria Research Report*, 7786.
      
1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) Approximate Bayesian approaches for reverse engineering biological networks. *Proceedings of the Kansas State University Conference on Applied Statistics in Agriculture*, Manhattan, Kansas.
      
1. **Rau, A.** (2008) Success of Volunteer Statistical Consulting Service Leads to Expanded Network. *The Statistical Consultant*, 25(1).
      
1. **Rau, A.** (2008) STATCOM Network Engages Growing Number of Student Volunteers. *Newsletter for the Section on Statistical Education*, 13(1).
      
1. **Rau, A.** (2008) Success of Statistical Service Leads to Expanded Network. *Amstat News*, April 2008.
