+++
# Date this page was created.
date = "2018-08-02"

# Project title.
title = "Multi-omic data integration"

# Project summary to display on homepage.
summary = "Integrate multi-omic data representing gene expression, methylation, copy number alterations, somatic mutations, and microRNA expression"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "edge_in_tcga_figure.png"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["integration"]

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
# [header]
# image = "headers/bubbles-wide.jpg"
# caption = "My caption :smile:"

+++

## Related software:

- [**padma**](https://github.com/andreamrau/padma): Pathway deviation scores using multiple factor analysis
- [**Regeneration Rosetta**](http://ls-shiny-prod.uwm.edu/rosetta/): An R/Shiny interactive web application to explore
regeneration-associated gene expression and chromatin accessibility
- [**maskmeans**](https://github.com/andreamrau/maskmeans): Multi-view aggregation/splitting K-means clustering algorithm.
- [**Edge in TCGA**](http://ls-shiny-prod.uwm.edu/edge_in_tcga/): An R/Shiny interactive web application for the exploration of drivers of gene expression in [The Cancer Genome Atlas](https://cancergenome.nih.gov/).


<img src="../../img/hex_integrative.png" alt ="" />


## Related publications:

* Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2019) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. *Annals of Applied Statistics* (accepted).
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01916941" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/441070" target="_blank" rel="noopener">
  ProdInra
</a>

* Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., **Rau, A.**, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Transcriptome and chromatin structure
annotation of liver, CD4+ and CD8+ T cells from four livestock species. *BMC Biology* (accepted). 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/441070" target="_blank" rel="noopener">
  ProdInra
</a>


* Dhara, S., **Rau, A.**, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. *Scientific Reports* 9:14198, doi: 10.1038/s41598-019-50485-6.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener">
 Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
 Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481438" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Dhara, S., Udvadia, A., and Auer, P. (2019) *Regeneration Rosetta*: An interactive web application to explore
regeneration-associated gene expression and chromatin accessibility. *G3: Genes|Genomes|Genetics*, doi: 10.1534/g3.119.400729..  
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener">
 Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481437" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Flister, M. J., Rui, H. and Livermore Auer, P. (2018) Exploring drivers of gene expression in The Cancer Genome Atlas. *Bioinformatics*, bty551, doi: https://doi.org/10.1101/227926.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener">
  Shiny app
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener">
 Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/452734" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.**, Manansala, R., Flister, M. J., Rui, H., Jaffr&eacute;zic, F., Lalo&euml;, D.&sup1;, and Auer, P. L.&sup1; (2019) Individualized multi-omic pathway deviation scores using multiple factor analysis *bioRxiv*, https://doi.org/10.1101/827022. 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/andreamrau/padma" target="_blank" rel="noopener">
  Software
</a>  
&sup1;These authors contributed equally to this work.




