+++
# Date this page was created.
date = "2018-08-02"

# Project title.
title = "Multi-omic data integration"

# Project summary to display on homepage.
summary = "Methodological contributions to integrative multi-omic analyses"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "integrative_illustrate.png"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["integration"]

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
# [header]
# image = "headers/bubbles-wide.jpg"
# caption = "My caption :smile:"

+++

## Related software:

* **padma**: Pathway deviation scores using multiple factor analysis, available at [BioC](https://bioconductor.org/packages/padma/)  
* **Regeneration Rosetta**: An R/Shiny interactive web application to explore regeneration-associated gene expression and chromatin accessibility, available as a [Shiny web app](http://ls-shiny-prod.uwm.edu/rosetta/)  
* **maskmeans**: Multi-view aggregation/splitting K-means clustering algorithm, available at [GitHub](https://github.com/andreamrau/maskmeans)  
* **Edge in TCGA**: An R/Shiny interactive web application for the exploration of drivers of gene expression in The Cancer Genome Atlas, available as a [Shiny web app](http://ls-shiny-prod.uwm.edu/edge_in_tcga/)  

<img src="../../img/hex_integrative.png" alt ="" />


## Related publications:

* **Rau, A.**, Manansala, R., Flister, M. J., Rui, H., Jaffrézic, F., Laloë, D., and Auer, P. L. (2020) Individualized multi-omic pathway deviation scores using multiple factor analysis. *Biostatistics*, kxaa029.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02968360v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/biostatistics/advance-article/doi/10.1093/biostatistics/kxaa029/5882080?guestAccessKey=9a15cdfb-100e-4478-8b68-e0a1f9e495f4" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/padma/" target="_blank" rel="noopener"> 
Software 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener"> 
Code 
</a>    
* Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. *Annals of Applied Statistics*, 14:2, 752-767.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789493" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://projecteuclid.org/euclid.aoas/1593449324" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener"> 
Code 
</a>    
* Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., **Rau, A.**, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. *BMC Biology*, 18:48.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02437599v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener"> 
Preprint 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="http://www.fragencode.org/" target="_blank" rel="noopener"> 
Website 
</a>  
* Dhara, S., **Rau, A.**, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. *Scientific Reports*, 9:14198.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02791772" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
* **Rau, A.**, Dhara, S., Udvadia, A., and Auer, P. (2019) Regeneration Rosetta: An interactive web application to explore regeneration-associated gene expression and chromatin accessibility. *G3: Genes|Genomes|Genetics*, 9(12): 3953-3959.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789080" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
* **Rau, A.**, Flister, M. J., Rui, H. and Livermore Auer, P. (2019) Exploring drivers of gene expression in The Cancer Genome Atlas. *Bioinformatics*, 35(1): 62-68.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02625326" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener"> 
Shiny app 
</a>  
