+++
# Date this page was created.
date = "2018-08-02"

# Project title.
title = "RNA-seq co-expression"

# Project summary to display on homepage.
summary = "Identify and visualize clusters of co-expressed genes from RNA-seq data"

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "coseq_hex.png"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["rnaseq"]

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
#[header]
#image = "headers/bubbles-wide.jpg"
# caption = "My caption :smile:"

+++

## Related software:

- [**coseq**](https://bioconductor.org/packages/coseq/): Co-expression analysis of sequencing data.
- [**ICAL**](https://github.com/Gallopin/ICAL): Model selection for model based clustering of annotated data.
- [**HTSCluster**](http://cran.r-project.org/web/packages/HTSCluster): Clustering high-throughput sequencing data with Poisson mixture models.


## Related publications:


* Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. *Journal of Applied Statistics*, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs" href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
  Code
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/481439" target="_blank" rel="noopener">
  ProdInra
</a>

* **Rau, A.** and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. *Briefings in Bioinformatics*, bbw128, https://doi.org/10.1093/bib/bbw128.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener">
  Preprint
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/413088" target="_blank" rel="noopener">
  ProdInra
</a>


* Mondet, F., **Rau, A.**, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018). Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. *BMC Genomics*, 19:328.
<a class="btn btn-primary btn-outline btn-xs" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/432918" target="_blank" rel="noopener">
  ProdInra
</a>


* Sauvage, C., **Rau, A.**, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Gl&eacute;min, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. *The Plant Journal* 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs" href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/393545" target="_blank" rel="noopener">
  ProdInra
</a>



* Gallopin, M., Celeux, G., Jaffr&eacute;zic, F., **Rau, A.** (2015) A model selection criterion for model-based clustering of annotated gene expression data. *Statistical Applications in Genetics and Molecular Biology*, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs" href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener">
  PDF
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener">
  Code
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/336851" target="_blank" rel="noopener">
  ProdInra
</a>



* **Rau, A.**, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. *Bioinformatics*, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs" href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener">
  Preprint
</a>
<a class="btn btn-primary btn-outline btn-xs" href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener">
  PDF
</a>
<a class="btn btn-primary btn-outline btn-xs" href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener">
  Code
</a>
</a> 
<a class="btn btn-primary btn-outline btn-xs" href="https://prodinra.inra.fr/record/311812" target="_blank" rel="noopener">
  ProdInra
</a>
