---
title: 2019 year in review
author: Andrea Rau
date: '2020-01-03'
slug: 2019-year-in-review
categories:
  - Year in review
tags: []
header:
  caption: ''
  image: ''
---

Well, here I am again, exactly one year after my last year-in-review post (apparently January 3 is peak procastination time when returning from the holiday break?).

The past year represented a whirlwind of change for me and my family. In the first half of 2019, I wrapped up the final few months of my [AgreenSkills+](https://www.agreenskills.eu/) sabbatical stay as a Visiting Scholar at [UWM](https://uwm.edu/) in Milwaukee, Wisconsin. My time at UWM was exciting, rich in new and continued collaborations, and a wonderful way to expand my research horizons while delving into the world of genomics applied to human health. I am especially grateful to my colleague and friend [Paul L. Auer](https://sites.uwm.edu/pauer/) for the wonderful welcome extended during my time there. In late June, my family and I moved back to France, starting with a month of couch-surfing in the Paris area and then with a move to the [Hauts-de-France](https://en.wikipedia.org/wiki/Hauts-de-France) region in northern France, where I have started splitting my time between two INRAE research units: a local plant genomics lab called [AgroImpact](http://www.hautsdefrance.inra.fr/Le-centre-Les-recherches/Les-unites-du-centre/AgroImpact) and my home lab in the [Animal Genetics and Integrative Biology](https://www6.jouy.inra.fr/gabi_eng/) (GABI) research unit<sup>1</sup>. That means that in the past year, I will have worked with human, animal, plant, and bacterial genomic data -- maybe I can find a way to add in fungal and viral genomics in 2020? :sweat_smile: (Also, as a side note: big surprise, it turns out that international moves are tiring and extremely disruptive to your personal and professional life!)

The past year allowed us to finish up and publish several of our ongoing projects focused on integrative multi-omics analysis, which has been a primary research interest for me lately:

- After a lot of hard work, in a collaboration with [Ava Udvadia](https://uwm.edu/biology/people/udvadia-ava/) at UWM, we published two companion papers based on short time-course study of matched RNA-seq and ATAC-seq data in a study on optic nerve regeneration in zebrafish. The [first](https://www.nature.com/articles/s41598-019-50485-6) discussed the comprehensive analysis and biological interpretation of the gene regulatory reprogramming during axon regeneration that we revealed, and the [second](https://www.g3journal.org/content/9/12/3953) showed how our Shiny app, the [Regeneration Rosetta](http://ls-shiny-prod.uwm.edu/rosetta/), could be used to deeply investigate our analyses and generate hypotheses in other organisms. 
- We were thrilled and very proud to see the [Fr-AgENCODE paper](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5) on livestock genome annotation get published after the heroic work put in during revisions by the two first authors. 
- In addition, the manuscript describing our [*maskmeans*](https://github.com/andreamrau/maskmeans) software to aggregate or split clusters using multi-view data was also accepted for publication after a (somewhat long but) helpful peer review process. 
- Finally, we were able to wrap up and submit our [work](https://www.biorxiv.org/content/10.1101/827022v2) on individualized pathway-specific deviation scores with our [*padma*](https://github.com/andreamrau/padma) software -- fingers crossed for next steps during the peer review process!

Otherwise, as with last year, I was quite lucky to have some more ongoing recent collaborations come to fruition, including 3 that were published and 2 that were submitted. I didn't do as much teaching as I have in past years (just a few hours of R instruction for a PhD-level class on physiological genomics at the [Medical College of Wisconsin](https://www.mcw.edu/), and for a local training course on RNA-seq data analysis), but I did get the very cool opportunity to teach a weeklong training course with my colleague Jordi Estell√© on 16S microbiome data analysis for our animal genetics colleagues at the [INRA French West Indies research center](http://www.antilles.inra.fr/en) in Guadeloupe. Since my return to France, I have been getting up to speed with the current state-of-the-art in genomic prediction models for the PhD project that I am currently supervising as part of the H2020 European-funded [GENE-SWitCH](https://t.co/vbpwkZNu38) grant.

Looking forward to 2020, I'll be continuing my deep dive into the integration of functional annotation information into Bayesian genomic prediction models, delving deeper into integrative multi-omic analyses (hello, partial triadic analysis!), and hopefully actually putting into action what I learned at a [UseR!2019](http://www.user2019.fr/) tutorial about adding [Travis Continuous Integration](https://travis-ci.org/) to my GitHub R packages. Now that the dust has finally started to settle after this recent period of upheaval, I am mostly hoping to see the path forward a bit more clearly!

--- 

<sup>1</sup> In a somewhat confusing twist, as of January 1, 2020 [INRA](http://www.inra.fr/) (the French National Institute for Agricultural Research) has [merged](https://agriculture.gouv.fr/linrae-verra-le-jour-le-1er-janvier-2020) with another French research institute called [Irstea](https://www.irstea.fr/fr) (the French National Research Institute of Science and Technology for Environment and Agriculture), and the combined research institute is now called [INRAE](https://www.inrae.fr) (the French National Institue for Agriculture, Food and Environment). My family already couldn't keep track of the difference between INRA and [Inria](https://www.inria.fr/en) (the French National Institute for Research in Digital Science and Technology, where I did my post-doc), so I haven't even attempted to explain the name change to them yet...


